lib/models/command.js

```diff
@@ -197,7 +198,7 @@ let Command = CoreObject.extend({
       return this._currentTask.onInterrupt();
     } else {
       // interrupt all external commands which don't use `runTask()` with an error
-      process.exit(1);
+      return RSVP.reject();
```

Comment:
@ro0gr when you say pass an exit code, do you mean use the exit code as the rejection value?  something like `RSVP.reject(1)`?

We will `process.exit(1)` if the cleanup promise (ie the at-exit work) rejects, but the command run promise isn't part of cleanup.

We could change the way the runner deals with command promises but of course we'll want to maintain the semantics that a SIGINT on `ember serve` (and similar) exits 0 while a SIGINT on most other things (eg `ember build`) exit non-zero.